dev-python/ciso8601/ciso8601-2.1.3.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+PYTHON_COMPAT=( python3_{7,8} )
```

Comment:
If it possible use PYTHON_COMPAT=( python3_{7..9} ) if it works with 9 too because now packages should works with it too and we prefer to test them against latest version of python

Reply:
I'll try to build with Python-3.9, but i'm not maintain ciso8601 sources.
and https://pypi.org/project/ciso8601/ - python3.9 not listed in supported versions

Reply:
Try it, if doesn't work try to figure out why and if it an upstream problem leave a comment